blint/cli.py

```diff
@@ -75,16 +77,20 @@ def main():
     if not src_dir:
         src_dir = os.getcwd()
         reports_base_dir = src_dir
+    elif len(src_dir)>1:
+        print("You must use the -o option to specify a reports output directory when scanning multiple sources.")
+        exit()
     else:
-        reports_base_dir = os.path.dirname(src_dir)
+        reports_base_dir = os.path.dirname(src_dir[0])
```

Comment:
Is src_dir always a list?

Reply:
> Is src_dir always a list?

Yes unless the target is not specified, in which case it will be NoneType and caught by the if statement which will use the current directory.